modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/plugin/ArtifactPluginDescriptorFactoryTestCase.java

```diff
@@ -154,6 +160,8 @@ public void pluginDescriptorNotFound() throws Exception {
   }
 
   @Test
+  @Feature(CLASSLOADING_ISOLATION)
+  @Stories({@Story(CLASSLOADER_CONFIGURATION_LOADER), @Story(CLASSLOADER_CONFIGURATION)})
```

Comment:
at the class level, this test is testing plugin descriptors, not classloader configuration